app/forms/invite_form.rb

```diff
@@ -0,0 +1,6 @@
+class InviteForm < Reform::Form
+  property :email
+  validates :email, :presence => true
+  validates :email, :email => true
+  validates_uniqueness_of :email
```

Comment:
I think these three validation lines can be condensed to one:

`validates :email, :uniqueness => true, :presence => true, :email => true`